design-documents/graph-ql/coverage/company.md

```diff
@@ -36,7 +36,10 @@ type Company @doc(description: "Company entity output data schema.") {
     ): CompanyRoles!  @doc(description: "Returns the list of defined roles at Company.")
     role(id: ID!): CompanyRole  @doc(description: "Returns company role by id.")
     acl_resources: [CompanyAclResource]  @doc(description: "Returns the list of all permission resources.")
-    hierarchy: CompanyHierarchyOutput  @doc(description: "Returns the complete data about company structure.")
+    structure(
+        rootID: ID = 0 @doc(description: "Tree depth to begin query")
```

Comment:
Seems like this field has wrong description

Reply:
That's the argument description